### PR TITLE
editors/code: Update supported debug engines in config

### PR DESCRIPTION
**Pull request by Master-Hash**
In PR #18265 and other we add more supported debug engine, but in VS Code's settings interface only CodeLLDB and MS C++ tools are shown available choices. This tiny PR updates the config to reflect it.

### CODE REVIEW

File: editors/code/package.json [hunk 1]
```diff
@@ -490,15 +490,19 @@
...
                             "vadimcn.vscode-lldb",
-                            "ms-vscode.cpptools"
+                            "ms-vscode.cpptools",
+                            "webfreak.debug"
                         ],
                         "default": "auto",
                         "description": "Preferred debug engine.",
                         "markdownEnumDescriptions": [
-                            "First try to use [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb), if it's not installed try to use [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools).",
+                            "First try to use [LLDB DAP](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.lldb-dap), if it's not installed try to use [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb), then use [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools), then [Native Debug](https://marketplace.visualstudio.com/items?itemName=webfreak.debug).",
```
Comment (lnicola) [inline editors/code/package.json:501]: Can you shorten this to something like "Use the first available extension out of ..."?

File: editors/code/package.json [hunk 2]
```diff
@@ -490,15 +490,19 @@
...
+                            "Use [LLDB DAP](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.lldb-dap)",
                             "Use [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)",
-                            "Use [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)"
+                            "Use [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)",
```
Comment (lnicola) [inline editors/code/package.json:504]:
```suggestion
                            "Use [C/C++ for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)",
```

**Comment by Master-Hash** [comment]
I believe it's ready (?
